nottrobin/documentation-builder
# Styling fixes following Lil's review

Styling fixes following Lil's review

Following: https://github.com/ubuntudesign/docs-vanilla-theme/issues/32

Do the following:

- Small screen: Fix extra spacing around header title
- Small screen: Make border below menu solid
- Largs & medium screens: Change navigation headers to 21px
- Decrease space below navigation headers to 1rem
- Small screen: Hide menu, provide CSS-only menu toggle button to show/hide it
- Small screen: Don't collapse the active menu section

QA
--

``` bash
git clone -b lil-styling-fixes https://github.com/nottrobin/documentation-builder.git  # This branch
git clone -b 1.0.0-structure https://github.com/nottrobin/ubuntu-core-docs.git  # A set of documentation to test with
pip3 install --user documentation-builder  # Install python modules
cd ubuntu-core-docs
../documentation-builder/bin/documentation-builder  # Build the docs
xdg-open build/en/index.html  # Check the built docs
```

Now check the above listed fixes are in place.
